field/src/field_types.rs

```diff
@@ -407,16 +404,14 @@ pub trait Field:
     }
 }
 
+pub trait PrimeField: Field {
+    fn to_biguint(&self) -> BigUint;
```

Comment:
Not strictly related to this PR, but while I'm here: For `u64` we have two variants of this method, `to_canonical_u64` and `to_noncanonical_u64`. The implementations of `to_biguint` all provide "`to_canonical_biguint`" functionality, so I think either this method should be called `to_canonical_biguint` or there should be some documentation for the caller so they know the result is reduced. I would tend to favour renaming the method so that it's similar to the other "upcast" methods, but I'm happy either way.

Reply:
Yeah good point - `to_canonical_biguint` seems fitting.